.env

```
@@ -0,0 +1,2 @@
+VITE_SUPABASE_URL=https://eqnrrvcmjobtuvbhuhau.supabase.co
+VITE_SUPABASE_ANON_KEY=sb_publishable_ctHtZ1C0eL34K05wGk06QQ_Xe6bEnd4
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SUPABASE_ANON_KEY key should go before the VITE_SUPABASE_URL key

(UnorderedKey)